lib/modulr/api/transactions_service.rb

```diff
@@ -3,10 +3,29 @@
 module Modulr
   module API
     class TransactionsService < Service
-      def history(account_id:)
-        response = client.get("/accounts/#{account_id}/transactions")
+      def list(account_id:, **opts)
+        response = client.get("/accounts/#{account_id}/transactions", build_query_params(opts))
         Resources::Transactions::Transactions.new(response, response.body[:content])
       end
+
+      private def build_query_params(opts) # rubocop:disable Metrics/AbcSize
+        same_name_params = [:credit, :type]
+        date_params = { to: :toTransactionDate, from: :fromTransactionDate, to_posted: :toPostedDate,
+                        from_posted: :fromPostedDate, }
+        mapped_params = {
+          source_id: :sourceId,
+          description: :q,
+          to_min: :minAmount,
+          from_max: :maxAmount,
```

Comment:
I find modulr naming more clear here and this...

```rb
client.transactions.list(from: Date.today -2, min_amount: 2_500, max_amount: 10_000)
```
... far more readable than this...

```rb
client.transactions.list(from: Date.today -2, to_min: 2_500, from_max: 10_000)
```